syntax = "proto3";
package task;




import "google/protobuf/struct.proto";

/*
CreateTask返回
*/
message CreateTaskResponse {
message Data {
    /*
    新建立的task_id
    */
    string id = 1 ;
}
        
    
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    结果信息
    */
    string msg = 2 ;
    /*
    返回数据
    */
    Data data = 3 ;
}

/*
CreateTaskApi返回
*/
message CreateTaskResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    task.CreateTaskResponse data = 4 ;
}

